src/react-components/settings-menu.js

```diff
@@ -215,22 +217,33 @@ export default class SettingsMenu extends Component {
                 </a>
               </div>
             </div>
-            {showRoomSettings ? (
+            {this.props.scene.is("entered") && showRoomSettings ? (
               <div className={rowClasses}>
                 <div className={styles.icon}>
                   <i>
                     <FontAwesomeIcon icon={faVideo} />
                   </i>
                 </div>
                 <div className={styles.listItem}>
-                  <div
-                    className={styles.listItemLink}
-                    onClick={() => {
-                      this.props.toggleStreaming();
-                    }}
-                  >
-                    <FormattedMessage id="settings.streamer-mode" />
-                  </div>
+                  {document.querySelector("#player-rig").components["player-info"].data.isStreaming ? (
```

Comment:
copying from other PR, this should go into a `TOOLS` section after `PROFILE` and `ROOM`